src/servers/exporter.ts

```diff
@@ -94,85 +94,99 @@ const coldWallets: { [key: string]: client.Gauge<string> } = {}
 const main = async () => {
   server.get("/metrics", async (req, res) =>
     asyncRunInSpan("metrics", {}, async () => {
-      const bosScore = await getBosScore()
-      bos_g.set(bosScore)
-
-      const { lightning, liabilities, bitcoin } = await getLedgerAccounts()
-      liabilities_g.set(liabilities)
-      lightning_g.set(lightning)
-      bitcoin_g.set(bitcoin)
-
-      try {
-        const { assetsLiabilitiesDifference, bookingVersusRealWorldAssets } =
-          await balanceSheetIsBalanced()
-        assetsLiabilitiesDifference_g.set(assetsLiabilitiesDifference)
-        bookingVersusRealWorldAssets_g.set(bookingVersusRealWorldAssets)
-      } catch (err) {
-        logger.error({ err }, "impossible to calculate balance sheet")
-      }
-
-      const {
-        total,
-        onChain,
-        offChain,
-        opening_channel_balance,
-        closing_channel_balance,
-      } = await lndsBalances()
-      lnd_g.set(total)
-      lndOnChain_g.set(onChain)
-      lndOffChain_g.set(offChain)
-      lndOpeningChannelBalance_g.set(opening_channel_balance)
-      lndClosingChannelBalance_g.set(closing_channel_balance)
-
-      const userCount = await User.countDocuments()
-      userCount_g.set(userCount)
-
-      for (const wallet of wallets) {
-        let walletId
+      await asyncRunInSpan("getBosScore", {}, async () => {
```

Comment:
Instead of awaiting each individual span, it could be more performant to kickoff each call and then do ```await Promise.all[... each span]```

Reply:
So we are already hammering the db  and request are very slow so I don't think this is a good idea (at least for now)